onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -5126,6 +5126,38 @@ def aten_scatter_add(
     return op.ScatterElements(self, index, src, axis=dim, reduction="add")
 
 
+@torch_op("aten::scatter_reduce", trace_only=True)
+def aten_scatter_reduce(
+    self: TReal,
+    dim: int,  # we have to use int here becuase ScatterElements() will use this attribute
+    index: TInt,
+    src: TReal,
+    reduce: str,
+    include_self: bool = True,  # pylint: disable=unused-argument
+):
+    """scatter_reduce.two(Tensor self, int dim, Tensor index, Tensor src, str reduce, *, bool include_self=True) -> Tensor"""
+
+    # FIXME(xiaowuhu): Support include_self == False
+    reduce_mode = {  # convert torch string name to onnx string name
+        "mean": "none",  # mean doesn't support in ORT
```

Comment:
```suggestion
        "mean": "none",
```

nit. Also: just to clarify, was it not supported in ORT or by ONNX? Should we open an issue in any case?

Reply:
Opened a feature request: https://github.com/onnx/onnx/issues/5100